content/docs/guides/vulnerability/scan-targets.md

```diff
@@ -125,12 +125,13 @@ syft alpine:latest -o json | grype
 
 **Read SBOM from file via stdin:**
 
+Note that Grype will not attempt to read from redirected stdin in interactive terminal sessions, the following is presented only to illustrate this as an option when working inside a scripted or other environment, where there is no interactive terminal session at play.
```

Comment:
we should probably change the example to:
```
cat sbom.json | grype
```
which should work in interactive sessions